src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -5,47 +5,45 @@ metaDescription: 'Manage New Relic and Azure integrations with agents installed
 
-* The .NET and Java APM agents
-* The infrastructure agent, for Windows and Linux systems
+## Notes on available New Relic features [#features]
 
-Other agents and integrations are available. You can browse them from within New Relic or on our [Instant Observability page](https://newrelic.com/instant-observability).
+* Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise edition. 
+* This services comes with our Data Plus option but not all Data Plus features are available. For example, HIPAA- and FedRAMP-compliant data is not yet available. For details, see [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus).
 
-Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise edition. 
-
-This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus) but not all Data Plus features are available. For details, see the [Data Plus docs](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus).
-
-[Learn more about features and benefits on the Azure Marketplace.](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg)
+For more about Azure-related features and billing, see [the Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg).
 
 ## Get the Azure Native New Relic Service [#get-it]
 
-For in-depth docs on how to set this up, see the [Microsoft Azure docs site](https://learn.microsoft.com/en-us/azure/partner-solutions/new-relic/new-relic-create), but here's a short summary of setup instructions. 
+For in-depth docs on how to set this up, see the [Microsoft Azure docs site](https://learn.microsoft.com/en-us/azure/partner-solutions/new-relic/new-relic-overview). Here's a short summary of how to set up the service. 
 
 First, you'll need to create a New Relic resource in the Azure Portal: 
 
 1. In the Azure Marketplace, subscribe to the Azure Native New Relic Service.
 2. In your Azure Portal, create a New Relic resource.
 3. Select **Create a New Relic resource** to create a new New Relic account, or you can **Link an existing New Relic resource** to link to an existing account.
 4. Complete the prompt that appears and select **Review + create**.
-5. Configure how your service reports metrics and logs in the **Resource configuration** tab.
+5. Configure how your service reports metrics and logs in the **Resource configuration** tab. 
 6. Verify that New Relic is receiving data in the **Monitored resources** tab of the Azure Portal. Resources with a green checkmark are sending data to New Relic.
```

Comment:
Before 6. Verify... there should be an item for installing infrastrucutre, .NET, and Java agents. We'll want our customers to do this before they head over to New Relic or they won't see any telemetry data from VMs or App Services.